### PR TITLE
Add institution model, dynamic forms, QA

### PR DESCRIPTION
**Pull request by Rytiggy**
This PR does a few things.

### Adds an institutional model 
```
  name: DS.attr('string'),
  AppName: DS.attr('string'),
  primaryColor: DS.attr('string'),
  secondaryColor: DS.attr('string'),
  tertiaryColor: DS.attr('string'),
  logo: DS.attr('string'),
  schema: DS.attr('string')
```
### Add dynamic forms 
Generalized Common and NIH forms to schemas that can be passed in to the metadata-from component

### CODE REVIEW

File: app/styles/app.scss [hunk 1]
```diff
@@ -211,11 +211,19 @@ body {
...
-      justify-content: center;
-      margin: auto;
-  }
+.border-none {
+  border: none!important;
+}
+.img-avatar {
+  width: 30px;
+}
+.cus-header {
```
Comment (cameronblandford) [inline app/styles/app.scss]: cus?

File: app/templates/application.hbs [hunk 2]
```diff
@@ -26,17 +26,41 @@
...
+          <li class="nav-item dropdown ml-auto">
+            <a class="nav-link" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
+              <img src="https://image.ibb.co/c8rxhx/bell_png_bell_icon_image_16624_512.png" style="width:20px" alt="admin@bootstrapmaster.com">
+              <span class="badge badge-pill badge-danger">3</span>
+            </a>
+            <div class="dropdown-menu dropdown-menu-right">
+              <div class="dropdown-header text-center">
+                <strong>Notifications</strong>
+              </div>
+              <a class="dropdown-item" href="#"><span class="badge badge-pill badge-danger">1</span>Submssion</a>
```
Comment (cameronblandford) [inline app/templates/application.hbs:40]: Spelling (Submssion -> submission)

File: app/templates/dashboard.hbs [hunk 3]
```diff
@@ -23,7 +15,9 @@
         </p>
         <div class="text-center">
           {{#link-to 'submissions' class='btn btn-primary mr-auto mt-1'}}View Submissions{{/link-to}}
-          {{#link-to 'submission.new' class='btn btn-primary mt-1'}}Add New Submission{{/link-to}}
+          {{#if session.isPI}}
```
Comment (cameronblandford) [inline app/templates/dashboard.hbs:18]: Nice!

File: app/index.html [hunk 4]
```diff
@@ -8,21 +8,22 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     {{content-for "head"}}
-    <link rel="stylesheet" href="{{rootURL}}coreUI.css">
     <link integrity="" rel="stylesheet" href="{{rootURL}}assets/vendor.css">
     <link integrity="" rel="stylesheet" href="{{rootURL}}assets/pass-ember.css">
     <link rel="icon" href="{{rootURL}}favicon.png">
-
+    <link rel="stylesheet" href="{{rootURL}}coreUI.css">
```
Comment (cameronblandford) [inline app/index.html:14]: 👍

**Review by cameronblandford — CHANGES_REQUESTED** [review]
Looks pretty good! Just give me responses to those comments when you get some time.

File: app/styles/app.scss (hunk 1 above)
Comment (cameronblandford) [reply app/styles/app.scss]: oooh custom - might be better with a different name, either written out or changed both work!

Comment (Rytiggy) [reply app/styles/app.scss]: Updated to custom-header although this class will be removed later on when building out the landing page further.

File: app/templates/application.hbs (hunk 2 above)
Comment (Rytiggy) [reply app/templates/application.hbs:40]: Good catch

**Pull request merged**